src/Accelerators/NNPA/Dialect/ZHigh/ZHigh.td

```diff
@@ -862,11 +862,16 @@ def ZHighStickifiedConstantOp:ZHigh_Op<"StickifiedConstant", [Pure]> {
   let summary = "ZHigh Stickified Constant operation";
   let description = [{
     This operator produces a constant tensor to store stickified data.
+    `value` attribute has original constant or stickified constant.
+    `stickified` attribute indicates the `value` is already stickified or not.
+    `allzero` attribute indicates the `value` will be all zero or not.
```

Comment:
Quick question" if it is `allzero` do we need to actually have the data, or it would be sufficient to have the shape as there is no need to store 10,000 times the value zero?

Reply:
Originally(in main branch) the value zeros are created and set in value attribute in ZHighStickifiedConstantOp. This consumes a lot of memory in some models. So, I created `allzero` attribute and set nullptr in value attribute in ZHighStickifiedConstantOp . Then, when generating LLVMIR, the value zeros are created and stored soon in the file. 
To answer your question, currently the value zeros are stored in the file. This PR delays the creation of the value zeros.

Reply:
If it is zero, you can use a splat dense attribute for `value` that stores one zero only and we don't need a new attribute `allzero`.

Reply:
I removed `allzero` attribute. Thanks!